swiftnav/src/time/utc.rs

```diff
@@ -57,30 +61,37 @@ impl UtcParams {
     }
 
     /// Modulo 1 sec offset from GPS to UTC \[s\]
+    #[must_use]
```

Comment:
Decided to look up why this is not the default generally and looks like I stumbled upon another extremely controversial topic in the Rust community lol. Kind of insane they don't have a way of setting this for all functions in the cargo.toml or something

Reply:
Yeah. @tzilist has suggested to start enabling the Clippy pedantic lints on more stuff. I don't think we've spent enough time to figure out if there are lints under that category that we want to blanket ignore. I think we're all happy to hear feedback if there are ones you think are just too noisy or are overly strict.

Reply:
Yeah no complaints just find it bizarre you need to cover all your functions with this attribute and there is no way to set it as a default for all functions and rather it be an opt out feature.